README.md

```diff
@@ -1 +1,198 @@
-# install-esp-idf-action
\ No newline at end of file
+# ESP-IDF Installation Action
+
+This GitHub Action automates the installation of the ESP-IDF framework on GitHub-hosted runners. It supports Windows, macOS (arm64 and Intel), and Linux (arm64 and x64) platforms, allowing you to set up ESP-IDF for your CI/CD workflows.
```

Comment:
Could you please add a link to https://github.com/espressif/esp-idf-ci-action with a comment saying that it should be used when you just need to build an app?